src/main.rs

```diff
@@ -21,7 +24,15 @@ use crate::tree::Tree;
 use crate::ui::{App, HeaderContext};
 
 // Forbid large data size to ensure TUI performance
-const MAX_DATA_SIZE: usize = 30 * 1024 * 1024;
+static MAX_DATA_SIZE: LazyLock<usize> = LazyLock::new(|| {
+    use std::str::FromStr;
+    let megabytes = std::env::var("OTREE_MAX_SIZE")
```

Comment:
Also, the fact, you are not using bytes but megabytes, let me think the variable name should reflect the unit is about megabytes